package.json

```diff
@@ -5,14 +5,17 @@
   "main": "app.js",
   "scripts": {
     "dev": "nodemon app.js",
-    "reset-db": "node ./db/seeds"
+    "reset-db": "node ./db/seeds",
+    "reset-db2": "node ./db/seeds2"
```

Comment:
Instead of creating a second file, I would just add this to the existing seeds file.